packages/semver/src/builders/version/plugin-adapter.ts

```diff
@@ -0,0 +1,22 @@
+import { Plugin, PluginOptions } from './plugin';
+
+export const SUPPORTED_SEMANTIC_RELEASE_PLUGINS = ['@semantic-release/npm'];
+
+export interface SemanticReleasePlugin extends Plugin {
+  addChannel?(options: PluginOptions): Promise<unknown>;
+}
+
+export class SemanticReleasePluginAdapter implements Plugin {
+  constructor(private _plugin: SemanticReleasePlugin) {}
+
+  publish(options: PluginOptions) {
+    this._plugin.addChannel(options);
```

Comment:
I think we should `concat` as  `addChannel` & `publish` are async.

Reply:
Temporary fixed using async / await, I will improve this by using `concat` when Observable results will be handled.